cmd/osm-controller/osm-controller.go

```diff
@@ -140,8 +140,8 @@ func main() {
 	}
 	log.Info().Msgf("Initial ConfigMap %s: %s", osmConfigMapName, string(configMap))
 
-	namespaceController := namespace.NewNamespaceController(kubeClient, meshName, stop)
-	meshSpec, err := smi.NewMeshSpecClient(*smiKubeConfig, kubeClient, osmNamespace, namespaceController, stop)
+	k8sclient := k8s.NewK8sClient(kubeClient, meshName, stop)
```

Comment:
I'd prefer renaming `k8s` to `kube` throughout this change.

Reply:
`kubeclient` already exists... hence the `k8s`. I do accept other suggestions though.

Reply:
This is okay then. Should avoid package aliasing to k8s unless there is a conflict.